GitCommands/Git/GitModule.cs

```diff
@@ -1334,21 +1334,34 @@ public string GetSubmoduleSummary(string submodule)
             return _gitExecutable.GetOutput(args);
         }
 
-        public void Reset(ResetMode mode, string? file = null)
+        public bool ResetAllChanges(bool resetAndDelete, bool onlyWorkTree = false)
```

Comment:
Could you please add xml-docs explaining each argument?

Reply:
done